lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala

```diff
@@ -61,7 +61,7 @@ case class SnapshotLsifCommand(
     } {
       val docPath = AbsolutePath
         .of(Paths.get(doc.getUri), sourceroot)
-        .toRealPath()
+        .normalize()
```

Comment:
This change was necessary to get the snapshots testing passing on macos. We don't run the CI on macos, but might want to reconsider that if we hit on more linux/macos issues like this.